sync/class.jetpack-sync-module-callables.php

```diff
@@ -86,6 +86,11 @@ public function maybe_sync_callables() {
 		if ( get_transient( self::CALLABLES_AWAIT_TRANSIENT_NAME ) ) {
 			return;
 		}
+
+		if ( ! defined( 'REST_API_REQUEST' ) ) {
+			define( 'REST_API_REQUEST', true );
```

Comment:
I am not sure this is the best way to go about this. Since it might cause some un expected consequences. 
Like additional code added to page load even if that is not desired. 

Is the expectation that the plugins only implement this filter when that constant is set to true?